README.md

```diff
@@ -28,28 +26,28 @@ const service = sdk.service((err, service) => {
 When running locally, you can specify a host and port to listen on by passing an options object with an optional host and port.  If no host/port is specified, localhost:10001 will be used:
 
 ```
-const service = sdk.service({ host: 'somehost', port: 7777 }, (err, service) => {
+sdk.service({ host: 'somehost', port: 7777 }, (err, flex) => {
   // code goes here
 };
 ```
 
 To run your code locally, execute `node .` in the root of your project.  Routes conform to the Kinvey Data Link specification.  
 
-## [DataLink framework](#datalink-framework)
+## [FlexData](#flex-data)
 
-The DataLink framework can be accessed via the sdk's `dataLink` property.
+The FlexData framework can be accessed via the sdk's `data` property.
 
 ```
-const dataLink = sdk.dataLink;
+const flexData = flex.data;
 ```
 
 ### Registering ServiceObjects
 
-Once you initialize the DataLink framework, you define your entites by defining ServiceObjects, and then wire up data access event handlers to those ServiceObjects.  To register a ServiceObject, use the `serviceObject` method of the `dataLink` framework:
+Once you initialize the FlexData framework, you define your entites by defining ServiceObjects, and then wire up data access event handlers to those ServiceObjects.  To register a ServiceObject, use the `serviceObject` method of the `data` framework:
```

Comment:
Should be either:

'use the `serviceObject` method of the `data` object:'

or

'use the `serviceObject` method of the `FlexData` framework:'